--- ../../Kernel-Simulator/public/Nitrocid/Kernel/KernelTools.cs.orig	2023-02-22 15:45:24.579641087 +0300
+++ ../../Kernel-Simulator/public/Nitrocid/Kernel/KernelTools.cs	2023-02-22 15:45:47.015641079 +0300
@@ -49,9 +49,9 @@
 
 #if SPECIFIERREL
 using static KS.ConsoleBase.Colors.ColorTools;
-using static KS.Misc.Notifications.Notifications;
+using static KS.Misc.Notifications.NotificationManager;
 using KS.Network;
-using KS.Network.Transfer;
+using KS.Network.Base.Transfer;
 #endif
 
 namespace KS.Kernel
